integration_tests/test_env_populator.py

```diff
@@ -12,7 +12,7 @@
 # create robyn.env before test and delete it after test
 @pytest.fixture
 def env_file():
-    CONTENT = """ROBYN_PORT=8080
+    CONTENT = """ROBYN_PORT=8081
```

Comment:
Why using 8081 here?

Reply:
@AntoineRR , as I wanted to test it on something other than the default port. Given that 8080 will be the default port from now on.

Reply:
>I just noticed there are a few places where port 5001 was used in the tests. Do you want to keep it like this?

Sure, this sounds like a good idea. 😄